.travis.yml

```diff
@@ -1,26 +1,18 @@
 language: php
 sudo: false
 php:
-  - 5.6
-  - 7.0
-  - 7.1
+#  - 5.6
+#  - 7.0
+#  - 7.1
   - 7.2
 
 env:
   global:
-    - CORE_BRANCH=master
```

Comment:
test with master as well

Reply:
I support only 10.0-10.99. Did not allow 10.1 yet.